src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -105,16 +105,28 @@ def generate_MCP_1_default_config(self, filename):
         dpa_3 = json.load(
           open(os.path.join('testcases', 'testdata', 'dpa_2.json')))
 
-        # Registration configuration for iteration 1
-        cbsd_records_iter_1 = {'registrationRequests': [device_1, device_2],
-                               'grantRequests': [grant_request_1, grant_request_2],
-                               'conditionalRequests': conditionals_iter_1
-                               }
-        cbsd_records_iter_2 = {'registrationRequests': [device_3, device_4],
+        # Registration and grant records for multiple iterations
+        cbsd_records_iter_1_domain_proxy_0 = {'registrationRequests': [device_1, device_2],
+                                              'grantRequests': [grant_request_1, grant_request_2],
+                                              'conditionalRegistrationData': conditionals_iter_1
+                                             }
+        cbsd_records_iter_1_domain_proxy_1 = {'registrationRequests': [device_3, device_4],
+                                              'grantRequests': [grant_request_3, grant_request_4]}
+ 
+        cbsd_records_iter_2_domain_proxy_0 = {'registrationRequests': [device_1, device_2],
```

Comment:
Why are you re-registering the device in the second iteration? I don't think this is allowed by the test spec.

Reply:
Updated,   new devices are used for the second iteration